src/main/java/longtimenosee/logic/commands/ViewIncomeCommand.java

```diff
@@ -12,7 +12,7 @@ public class ViewIncomeCommand extends Command {
 
     public static final String COMMAND_WORD = "viewIncome"; //should I change to refresh?
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + "<YEAR>" + ": chooses the year for viewing income";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " <year>" + ": chooses the year for viewing income";
     private final int targetYear;
 
     public ViewIncomeCommand(int targetYear) {
```

Comment:
i noticed in your execute method in ViewIncomeCommand that you used a variable "i", maybe you could use a more descriptive term such as income?

Reply:
Rectified! Thank you!